packages/mask/background/database/avatar-cache/avatar.ts

```diff
@@ -49,7 +49,6 @@ export const queryAvatarsDataURL: (
  */
 
 export async function storeAvatar(identifier: IdentifierWithAvatar, avatar: ArrayBuffer | string): Promise<void> {
-    if (identifier instanceof ProfileIdentifier && identifier.isUnknown) return
```

Comment:
Will deleting this cause any errors? cc @lelenei @guanbinrui

Reply:
Oh wait, I made the wrong change.

Reply:
Hmm, no, this is correct. Because the only case we want to exclude is the ProfileIdentifier.unknown, now it is replaced with null.